src/main/groovy/edu/oregonstate/mist/locations/core/FacilLocation.groovy

```diff
@@ -55,18 +55,16 @@ class FacilLocation extends BaseType {
 
     @JsonIgnore
     String getPrettyCampus() {
-        if (campus == null) {
-            return null
-        }
-
-        if (campus.equalsIgnoreCase("CASCADESCAMPUS")) {
-            "Cascades"
+        if (!campus) {
```

Comment:
I'm fine with changing `if (campus == null)` to `if (!campus)`, but one thing to note is how this could change the method. Before, if a campus was just `''`, it would get assigned the `other` campus because `'' != null`. By doing `!campus`, a value of `''` would now be assigned `null` from this method. However, since this method is used from the string directly from the database, I don't think Oracle differentiates between null and an empty string. This is just somewhat of a tangent, but either way, I think `(!campus)` is better since I don't think we want an empty string being assigned 'Other'.

Reply:
Ok, good to know. I agree that for the case of an empty string that the campus should be `null` instead of `"Other"`.